tests/test_it.py

```diff
@@ -69,6 +69,40 @@ def test_b():
     out.assert_outcomes(passed=2, failed=0)
 
 
+def test_it_reuses_the_prev_run_seed(ourtestdir):
+    """
+    Run a test that exercises the randomly-repeat-last option.
+    """
+    ourtestdir.makepyfile(
+        test_one="""
+        def test_a():
+            pass
+        """
+    )
+    out = ourtestdir.runpytest('--randomly-seed=33')
+    out.assert_outcomes(passed=1, failed=0)
+    out.stdout.fnmatch_lines(['Using --randomly-seed=33'])
+    ourtestdir.makepyfile(
+        test_one="""
+        def test_a():
+            pass
+        """
+    )
+    out = ourtestdir.runpytest('--randomly-repeat-last')
+    out.assert_outcomes(passed=1, failed=0)
+    out.stdout.fnmatch_lines(['Using --randomly-seed=33'])
+    ourtestdir.makepyfile(
+        test_one="""
+        def test_a():
+            pass
+        """
+    )
+    out = ourtestdir.runpytest()
+    out.assert_outcomes(passed=1, failed=0)
+    for line in out.outlines:
+        assert 'Using --randomly-seed=33' not in line
```

Comment:
This final part of the test would fail depending on the environment, if the default seed happened to be 33. Practically this is very unlikely, the computer's clock would have to be at 33 seconds past unix epoch, but it still isn't good practice to have an intermittently failing test, nor to have a 'negative assertion', so I'm removing it